netcdf/src/error.rs

```diff
@@ -145,9 +145,9 @@ impl fmt::Display for Error {
                 "buffer size mismatch, has size {actual}, but needs size {wanted}",
             ),
             Self::TypeMismatch => write!(f, "netcdf types does not correspond to what is defined"),
-            Self::TypeUnknown(t) => write!(f, "netcdf type {} is not known", t),
+            Self::TypeUnknown(t) => write!(f, "netcdf type {t} is not known"),
             Self::AlreadyExists => write!(f, "variable/group/dimension already exists"),
-            Self::NotFound(x) => write!(f, "Could not find {}", x),
+            Self::NotFound(x) => write!(f, "Could not find {x}"),
```

Comment:
These are inconsistent, but I think most errors will display themselves with a lower-case letter.
```suggestion
            Self::NotFound(x) => write!(f, "could not find {x}"),
```